sopel/bot.py

```diff
@@ -1339,7 +1374,8 @@ def reply(self, message, destination=None, reply_to=None, notice=False):
             :meth:`sopel.bot.Sopel.reply`
         """
         if destination is None:
-            destination = self._trigger.sender
+            destination = self.default_destination
+
```

Comment:
Quibble: I like the extra space here that separates the choosing of a default value from the action of this method, but would prefer that `notice()` and `action()` have the same style as `reply()` (i.e. a separating blank line)